src/room.ts

```diff
@@ -356,26 +390,59 @@ export class Room extends EventEmitter implements Sayable {
   }
 
   /**
-   * NickName / DisplayName / RemarkName of member
+   * find member by `nick`(NickName) / `display`(DisplayName) / `remark`(RemarkName)
+   * when use member(name:string), find member by nickName by default
    */
-  public member(name: string): Contact | null {
-    log.verbose('Room', 'member(%s)', name)
+
+  public member(filter: MemberQueryFilter): Contact | null
+  public member(name: string): Contact | null
+
+  public member(queryArg: MemberQueryFilter | string): Contact | null {
+    if (typeof queryArg === 'string') {
+      log.warn('Room', 'member(%s) DEPRECATED, use member(queryArg: MemberQueryFilter) instead.', queryArg)
+      return this.member({remark: queryArg}) || this.member({display: queryArg}) || this.member({nick: queryArg})
+    }
+
+    log.verbose('Room', 'member({ %s })'
```

Comment:
use `log.silly` instead of `verbose` here.

`verbose` should be used only once per function, usually at the beginning of the function, to log the function is run and all parameters it has received.

Any other log should be `silly`